Source/Service/Models/Identity.swift

```diff
@@ -8,21 +8,21 @@
 import Foundation
 
 public struct CotterIdentity: Codable {
-    var identifier: Identifier
-    var token: Token
+    public var identifier: Identifier
+    public var token: Token
 }
 
 public struct Identifier: Codable {
-    var id:String
-    var createdAt:String
-    var updatedAt:String?
-    var deletedAt:String?
-    var identifier: String
-    var identifierType: String
-    var publicKey: String
-    var deviceType: String
-    var deviceName: String
-    var expiry: String
+    public var id:String
+    public var createdAt:String
+    public var updatedAt:String?
+    public var deletedAt:String?
+    public var identifier: String
+    public var identifierType: String
+    public var publicKey: String
+    public var deviceType: String
+    public var deviceName: String
+    public var expiry: String
```

Comment:
adding public scope means that we're exposing all these properties to the clients, meaning they're gonna know all these structs and properties within the structs and they can access them directly. i don't know if it could expose vulnerabilities in the code, maybe/not. i’m fine with it if we want to expose these, but sometimes i found all these public scopes added in unnecessary places, i.e. structs/classes/functions/properties that aren't supposed to be known by the clients.

so just for reference, please don't add them arbitrarily, we don't wanna give out too much information. [not applicable for this one, but just for reference] at least for properties (`var`'s or `let`'s), one alternative/workaround is to remove the public and create an `init()` of the class. this way we'd restrict the clients to directly access the properties.